pre_built_networks/1class8/guessing_config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "args" : {
+    "arch_file" : "arch.txt",
+    "weight_file" : "weights.h5",
```

Comment:
Can you change this to weights.h5.final to match the file that is uploaded? When I try to use it on my machine, it complains about this file not existing.

Reply:
I had the weight file name in the config different from what was checked in because if anyone runs the training phase, that will just overwrite the final weights file. And all the configs have the same 'weights.h5' name because the same weights will be shared among them. Do you want me to rename the 'weights.h5.final' to 'weights.h5', risking that the final weights might be overwritten?

Reply:
I'm not sure I understand. Doesn't the overwriting only happen in the training config? So could we change the "weight_file" in the guessing config to be "weights.h5.final" without there being a danger of people overwriting the weights?